build-system/babel-plugins/babel-plugin-transform-jss/index.js

```diff
@@ -86,12 +164,43 @@ module.exports = function ({template, types: t}) {
       },
 
       CallExpression(path, state) {
+        const callee = path.get('callee');
+
+        // Replace users that import JSS.
+        const importDeclaration = findImportDeclaration(callee, 'useStyles');
+        if (importDeclaration) {
+          const {parentPath} = path;
+          // Such as: useStyles().myClassName
+          if (replaceMemberExpression(importDeclaration, path, parentPath)) {
+            return;
+          }
+          // Such as: const classes = useStyles();
+          if (parentPath.isVariableDeclarator()) {
```

Comment:
Need to check that it's not a destructure:

```suggestion
          if (parentPath.isVariableDeclarator() && parentPath.get('id').isIdentifier()) {
```